modules/react/_examples/stories/examples/SidePanelWithNavigation.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import {styled, StyledType} from '@workday/canvas-kit-react/common';
+import {colors, space, gradients, type} from '@workday/canvas-kit-react/tokens';
+import {SidePanel, useSidePanel} from '@workday/canvas-kit-preview-react/side-panel';
+import {Flex, Box} from '@workday/canvas-kit-react/layout';
+import {BodyText, Subtext} from '@workday/canvas-kit-react/text';
+import {Expandable} from '@workday/canvas-kit-labs-react/expandable';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {birthdayIcon, ribbonIcon} from '@workday/canvas-system-icons-web';
+
+const StyledPanel = styled(SidePanel.as('nav'))({
+  backgroundColor: colors.soap200,
+  zIndex: 1,
+});
+
+const StyledExpandable = styled(Expandable.Target)({
+  borderRadius: 0,
+  ':hover': {
+    backgroundColor: colors.soap500,
+  },
+});
+
+const StyledLink = styled(Subtext.as('a'))<StyledType>({
+  color: colors.blackPepper300,
+  ':hover': {
+    backgroundColor: colors.soap500,
+  },
+});
+
+const cakes = [
+  {value: 'Anniversary', linkValue: '#AnniversaryCakes'},
+  {value: 'Birthday', linkValue: '#BirthdayCakes'},
+  {value: 'Graduation', linkValue: '#GraduationCakes'},
+  {value: 'Wedding', linkValue: 'WeddingCakes'},
+];
+
+const bestsellers = [
+  {value: 'Mango Coco Sago', linkValue: '#LemonRhubarbBars'},
+  {value: 'Matcha Creme Pie', linkValue: '#MatchaCremePie'},
+  {value: 'Key Lime Cheesecake', linkValue: '#KeyLimeCheesecake'},
+  {value: 'Tiramisu', linkValue: '#Tiramisu'},
+];
+
+const gettingStarted = [
+  'Extended Systems',
+  'Canvas Glossary',
+  'Frequently Asked Questions',
+  'Canvas Support',
+];
+
+export const WithNavigation = () => {
+  const {labelProps} = useSidePanel();
+
+  return (
+    <Flex height={800} position="relative" backgroundColor="frenchVanilla100">
+      <StyledPanel aria-label="Main navigation" touched={true} variant="alternate">
```

Comment:
Since this StyledPanel component is already rendering a `<nav>` element, I would avoid using the term "Navigation" in aria-label strings. (In general, avoid writing the implied `role` into `aria-label` strings.)

Second, "main" could be confusing when a `<main>` region is used for the body content. I think for the purposes of this example, omitting `aria-label` might be best. 

If you want to keep it, I'd consider using a term that describes the purpose of the navigation. What do you think about "Dessert Recipes" instead? 

<img width="1627" alt="nav tag aria label" src="https://github.com/Workday/canvas-kit/assets/40372497/6dadfef4-bf5e-407e-8d6e-85252899a28c">

Reply:
removed!